Godot_test/ClientScripts/PlayerStatsMenu.cs

```diff
@@ -0,0 +1,47 @@
+using Godot;
+using System;
+
+public partial class PlayerStatsMenu : Control
+{
+  public bool isOn = false;
+
+  public void UpdateStatsData()
+  {
+	var hero = Game.hero.HeroTile;
+
+	var name = (Label)GetNode("General/Name/Value");
+	name.Text = hero.Name;
+	var gold = (Label)GetNode("General/Gold/Value");
+	gold.Text = hero.Gold.ToString();
+	var level = (Label)GetNode("General/Level/Value");
+	level.Text = hero.Level.ToString();
+	var exp = (Label)GetNode("General/Exp/Value");
+	exp.Text = Math.Round(hero.Experience).ToString() + "/" + Math.Round(hero.NextLevelExperience).ToString();
+
+	var availablePoints = (Label)GetNode("Attributes/AvailablePoints");
+	availablePoints.Text = "Expirience Points to assign: " + hero.LevelUpPoints.ToString();
+	var health = (Label)GetNode("Attributes/Health/Value");
+	health.Text = hero.Stats.GetCurrentValue(Roguelike.Attributes.EntityStatKind.Health).ToString() + "/" + hero.Stats.GetTotalValue(Roguelike.Attributes.EntityStatKind.Health);
```

Comment:
1)
1)
1) 

1)
Way too much code.

Add a method:
string GetDisplayedStat(EntityStatKind esk, Hero ent)
var value = ent.Stats.GetStat(kind).Value;

return value.Current/value.Nominal;

2. (optional)
If you knew which label is for which attribute (have naming convention) you could do it in the loop.

List<Roguelike.Attributes.EntityStatKind> stats = new {Roguelike.Attributes.EntityStatKind.Health, Roguelike.Attributes.EntityStatKind.Strength, ...}

foreach(var stat in stats)
{
  var label = getLabel(stat );
  label.Text =  GetDisplayedStat(stat );
}